pkg/api/controllers/account_controller.go

```diff
@@ -38,11 +62,35 @@ func (ctl *AccountController) CountAccounts(c *gin.Context) {
 func (ctl *AccountController) GetAccounts(c *gin.Context) {
 	l, _ := c.Get("ledger")
 
+	balance := c.Query("balance")
+	if balance != "" {
+		if _, err := strconv.ParseInt(balance, 10, 64); err != nil {
+			ResponseError(
+				c,
+				ledger.ValidationError{
+					Msg: "invalid parameter 'balance', should be a number"})
+			return
+		}
+	}
+
+	balanceOperator := c.Query("balance_operator")
+	if balanceOperator != "" {
+		if !BalanceOperator(balanceOperator).IsValid() {
+			ResponseError(
+				c,
+				ledger.ValidationError{
+					Msg: "invalid parameter 'balance_operator', should be one of 'e, gt, gte, lt, lte'"})
+			return
+		}
+	}
+
 	cursor, err := l.(*ledger.Ledger).GetAccounts(
 		c.Request.Context(),
 		query.After(c.Query("after")),
 		query.Address(c.Query("address")),
 		query.Metadata(c.QueryMap("metadata")),
+		query.Balance(balance),
+		query.BalanceOperator(balanceOperator),
```

Comment:
Maybe make query.BalanceOperator() accept a BalanceOperator type instead of a string.